metadata-ingestion/src/datahub/ingestion/source/data_lake/__init__.py

```diff
@@ -310,25 +315,50 @@ def get_table_schema(
         )
         dataset_snapshot.aspects.append(dataset_properties)
 
-        column_fields = []
+        if file_path.startswith("s3a://"):
```

Comment:
Is this the only applicable prefix or are there more?

Reply:
This is the only applicable one at this point — we resolve the other S3 prefixes before ingesting